x/datadeal/testutil/suite.go

```diff
@@ -42,3 +42,19 @@ func (suite *DataDealBaseTestSuite) MakeNewDataSale(sellerAddr sdk.AccAddress, v
 		DeliveryTallyResult:     nil,
 	}
 }
+
+func (suite *DataDealBaseTestSuite) MakeNewDataSaleDeliveryVoting(sellerAddr sdk.AccAddress, verifiableCID string) *types.DataSale {
+	return &types.DataSale{
```

Comment:
Is this method return `types.DataSale`? I think this should be return `types.DataDeliveryVote`.
Also, how about change name to `MakeNewDataDeliveryVote`?

Reply:
Ah, I want to make `DataSale` that `status` is `DATA_SALE_STATUS_DELIVERY_VOTING_PERIOD` not `DATA_SALE_STATUS_VERIFICATION_VOTING_PERIOD`. If a function to change `status` is added later, I will delete it.

Reply:
Ah, I got it.